hugo/layouts/partials/header.html

```diff
@@ -1,30 +1,74 @@
-<nav class="fullWidth topbar {{ .Params.topbar }}" id="topBar">
+{{ $baseurl := "//sourced.tech" }}
+
+<nav class="topbar" id="topbar">
     <div>
-        <a href="{{ .Site.BaseURL }}" alt="source{d}">
-            <div class="logo"></div>
+        <a href="{{ $baseurl }}" alt="source{d}">
+            <div class="logo"><img src="{{ "/img/logos/source-logo-white.svg" | absURL }}" alt="source{d}" /></div>
         </a>
-        <div class="menu" id="menu">
+        <div class="topbar__menu" id="menu">
             <ul>
                 <li {{ if eq .URL "/" }}class="active"{{ end }} title="homepage">
-                    <a href="{{ .Site.BaseURL }}">home</a>
+                    <a href="{{ $baseurl }}" title="Home">Home</a>
                 </li>
                 <li {{ if eq .URL "projects" }}class="active"{{ end }}>
-                    <a href="{{ .Site.BaseURL }}/projects" title="Projects">projects</a>
+                    <a href="{{ $baseurl }}/projects" class="topbar__menu--projects" id="submenu-toggle" title="Projects">
```

Comment:
I maybe missed a thing, so I need to ask...
Why didn't you used `absURL` as done in the image src?
It think it could be:
`{{absURL projects}}` instead of `{{ $baseurl }}/projects`
and
`{{absURL /}}` instead of `{{ $baseurl }}`